day_7/Fizzbuzz.js

```diff
@@ -0,0 +1,17 @@
+for (var i = 0; i < 101; i++ ) {
+
+  var divideByThree = i % 3;
```

Comment:
One quick note, this reads like a method to me. For vars, I like them in past tense. ie. divided by 3